### ISSUE TITLE
IP Loggers/Grabbers

### ISSUE THREAD

**Issue by Bluscream** [AdguardTeam/AdguardFilters#6623]
If there's already a discussion/issue about that please link me to it. Thanks :)

With the growing abilities for everyone to DDoS anyone for free ip loggers are becoming a real social engineering threat.

You might ask: "why are these any different than tracking services like ip-api.info which are already blocked by ag"
well, the trackers like ip-api are usually embed in websites which means they can be easily discovered by the AG team. Different with ip logger links, they are provided directly (tho obfuscated through a chain of url shorteners) by the "attacker" to the victim. So someone has to report them and i'm sure that the first place that comes to his mind won't be his adblocker. He most likely will complain on reddit where everyone will comment something like "these scriptkiddies" but nothing really changes.

If AG had a filterlist for those nasty grabber services (which should also be included in AG dns) that would make it easy to tell the victim that he almost got "scammed" by someone and ag protected him. Also i would love to have this list enabled by default because most non techy users that would fall for an iplogger more easily don't usually bother scraping through the list of available filters and looking up what an iplogger even is.

The easiest way to find such services is by simply [google'ing them](https://www.google.com/search?q=ip+logger) (thats what the "attackers" do aswell)
Of course there are some templates for self-hosted iploggers but setting that up would require atleast some skills. The "attacker" usually has none :^)

I would gadly PR such a list today if you tell me where do you want that file to be stored in your repo ^~^

---
Sent from my Lenovo TAB 2 A10-70L running Android 6.0 via OctoDroid v4.2.0

**Comment by ameshkov** [comment]
> Different with ip logger links, they are provided directly (tho obfuscated through a chain of url shorteners) by the "attacker" to the victim

It requires "strict" blocking for these domains. We may need to introduce a special modifier that will disable our "block only sub-requests" default behavior.

Other than that -- we do not need a separate filter for them, it seems to me that these domains fit spyware filter just okay.

**Comment by Bluscream** [comment]
A seperate filter would allow other extensions/userscripts/addons to use it tho. For example i'd make a teamspeak plugin for it.

**Comment by Bluscream** [comment]
Why was it closed?

**Comment by Bluscream** [comment]
@Alex-302 @ameshkov

**Comment by Alex-302** [comment]
it was moved to the new issue
https://github.com/AdguardTeam/CoreLibs/issues/70

**Comment by Bluscream** [comment]
Ahh ok thanks :)
